app/assets/stylesheets/widgets/primary_column/wysiwyg.scss

```diff
@@ -4,4 +4,12 @@
   .content > p:last-child {
     margin-bottom: 0;
   }
-}
\ No newline at end of file
+
+  #map_frame {
+    min-height: 59vh;
+
+    @include bp(wildcard-max, 767px) {
```

Comment:
I saw these two comments in the mixins file that stated I should use the `bp(wildcard-min, 767px)` when setting the breakpoint for medium devices. 

`// There are some obvious problems with this. For example, we have a breakpoint defined
// at 780px for m(edium). But our designs target 768px and in neat/_settings_grid.scss,
// you'll find this setting: $medium-screen: 768px;`

`// This should probably not be used (see note above). Leaving it as is because
  // not sure what changing it would break.
  // Use this instead: bp(wildcard-min, 767px)
       @if $point == m { // Medium
		@media (min-width: 780px) { @content; }
	}`

So should I change this to what you stated?

Reply:
Yes. It looks like Tom added those comments a while ago, and he didn't give an explanation as to why he believes we shouldn't be using them this way. `@include media([variable])` is used widely across all the stylesheets, and we continued to use it after he added those comments and I never heard him mention it. So when we refactor the stylesheets I might remove those comments.

Reply:
Sounds good, I've updated the media query to use the $medium-screen variable.